ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -55,10 +55,10 @@ import           Ouroboros.Consensus.Ledger.SupportsPeerSelection
 import           Ouroboros.Consensus.Ledger.SupportsProtocol
 import           Ouroboros.Consensus.Mempool
 import qualified Ouroboros.Consensus.MiniProtocol.BlockFetch.ClientInterface as BlockFetchClientInterface
-import           Ouroboros.Consensus.MiniProtocol.ChainSync.Client.InFutureCheck
-                     (HeaderInFutureCheck)
 import           Ouroboros.Consensus.MiniProtocol.ChainSync.Client
                      (ChainSyncClientHandle)
+import           Ouroboros.Consensus.MiniProtocol.ChainSync.Client.InFutureCheck
+                     (HeaderInFutureCheck)
```

Comment:
I suppose this is `stylish-haskell` at play? I feel this is completely irrelevant to us. Any idea how that made it into `genesis/milestone6`? Is `master` broken? Can we just ignore `stylish-haskell` now?

Reply:
uh, this is based on `port-gdd`

Reply:
after having rebased, the change is gone.